README.md

```diff
@@ -46,11 +46,11 @@ keda-hpa-dummy   Deployment/dummy   <unknown>/10 (avg)   1         4         0
 
 To scale up we have to populate the Redis queue. To do this we can use the helper app:
 ```shell script
-kubectl exec -it $(k get pods | grep "server" | cut -f 1 -d " ") app redis publish
+kubectl exec $(kubectl get pods | grep "server" | cut -f 1 -d " ") -- keda-talk redis publish
 ```
 and to scale down:
 ```shell script
-kubectl exec -it $(k get pods | grep "server" | cut -f 1 -d " ") app redis drain
+kubectl exec $(kubectl get pods | grep "server" | cut -f 1 -d " ") -- keda-talk redis drain
```

Comment:
Can you explain the `keda-talk` part?

Reply:
Just like the MySQL example it's used communicate with the app in order to add items to the redis list or drain it.

Reply:
@genesysflow I think the example for MySQL is wrong - there's no `keda-talk` app deployed. I must have missed that when accepting last change.

Reply:
The entry point in Dockerfile is no longer app since [this](https://github.com/turbaszek/keda-example/commit/e3330467ccc33d1db4294d8fb02c4b20c6b91160) commit so instead of the `app` we must use `keda-talk`